### ISSUE TITLE
CPM training loss saturate very early

### ISSUE THREAD

**Issue by soon-yau** [edvardHua/PoseEstimationForMobile#54]
With default setting of mv2_cpm.cfg running on single GPU, the loss drop very quickly to about 1000, and last_heat_loss to 160 in about 300 steps and then it losses don't improve anymore. On the other hand, the hourglass model seems to be training better (currently loss=350 after about 1 epoch).

Any idea why CPM might struggle to train?
